service/src/test/java/tds/assessment/services/impl/AssessmentServiceImplIntegrationTest.java

```diff
@@ -0,0 +1,48 @@
+package tds.assessment.services.impl;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+
+import java.util.Arrays;
+import java.util.Optional;
+
+import tds.assessment.Algorithm;
+import tds.assessment.Assessment;
+import tds.assessment.Segment;
+import tds.assessment.repositories.AssessmentQueryRepository;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+@RunWith(SpringJUnit4ClassRunner.class)
+@SpringBootTest
+public class AssessmentServiceImplIntegrationTest {
```

Comment:
Looking at the pom.xml integration tests must be plural in the name.  So this should be renamed `AssessmentServiceImplIntegrationTests`.  You will notice this doesn't execute via Maven without the change.